releases/catalog-service-monitor.yaml

```diff
@@ -7,17 +7,18 @@ items:
   metadata:
     name: stakater-nordmart-catalog-dev-monitor
     labels:
-      release: stakater-monitoring-prometheus-operator
+      release: stakater-monitoring-po
```

Comment:
Same here

Reply:
currently in our monitoring stack Prometheus operator has been named like this and this needs to match that release name. Secondly this was named as po intentionally to make the name a little short.